nginx.master.conf

```diff
@@ -285,35 +285,42 @@ http {
             proxy_pass http://log/;
         }
 
-        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)/logs/v1/(?<url>.*)$ {
+        location /system/v1/metrics/ {
+            access_by_lua 'auth.validate_jwt_or_exit()';
+
+            include common/proxy-headers.conf;
+            proxy_pass http://metrics/;
+        }
+
+		# Ordering is important for these two endpoints!!!
+        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)(?<type>(/metrics/v0|/logs/v1))(?<url>.*)$ {
```

Comment:
@darkonie can you double check this please? RE: your comment yesterday in the day2ops channel.